### ISSUE TITLE
EventEmitter memory leak on 11+ resources

### ISSUE THREAD

**Issue by DuncanFenning** [holidayextras/jsonapi-store-relationaldb#8]
When instantiating a new RelationalStore for 11 or more resources, you get a node warning:

```
node) warning: possible EventEmitter memory leak detected. 11 listeners added. Use emitter.setMaxListeners() to increase limit.
Trace
    at process.EventEmitter.addListener (events.js:160:15)
    at process.on.process.addListener (node.js:773:26)
    at ConnectionManager (/Users/duncan.fenning/git/hapi/node_modules/jsonapi-store-relationaldb/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:45:11)
    at new ConnectionManager (/Users/duncan.fenning/git/hapi/node_modules/jsonapi-store-relationaldb/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:12:29)
    at new MysqlDialect (/Users/duncan.fenning/git/hapi/node_modules/jsonapi-store-relationaldb/node_modules/sequelize/lib/dialects/mysql/index.js:12:28)
    at new Sequelize (/Users/duncan.fenning/git/hapi/node_modules/jsonapi-store-relationaldb/node_modules/sequelize/lib/sequelize.js:203:20)
    at SqlStore.initialise (/Users/duncan.fenning/git/hapi/node_modules/jsonapi-store-relationaldb/lib/sqlHandler.js:35:20)
    at Object.jsonApi.define (/Users/duncan.fenning/git/hapi/node_modules/jsonapi-server/lib/jsonApi.js:57:29)
    at Object.<anonymous> (/Users/duncan.fenning/git/hapi/jsonApi/trips.js:6:9)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Array.forEach (native)
    at module.exports (/Users/duncan.fenning/git/hapi/jsonApi.js:63:6)
    at Object.<anonymous> (/Users/duncan.fenning/git/hapi/server.js:140:24)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
```

**Comment by theninj4** [comment]
This might be fixed by #22

**Comment by pmcnr-hx** [comment]
This has now been fixed by supporting multiple resources in a single database in `v1.1.0` so closing.
